File: insee-promotion-admin-client/app/components/enum/ImageStatus.js
export class  ImageStatus {
    constructor(status) {
        this.status = status
    }

    getStatus() {
        return this.status;
    }

    static findByStatus(status) {
        switch(status) {
            case 1 : return WAITING_APPROVAL;
            case 2 : return APPROVED;
            case 3 : return REJECTED;
        }
    }

    static getColor(status) {
        switch(status) {
            case 1: return '#fff';
            case 2: return '#28a745';
            case 3: return '#b71c1c'
        }
    }
    
}
export const WAITING_APPROVAL = new ImageStatus(1);
export const APPROVED = new ImageStatus(2);
export const  REJECTED = new ImageStatus(3);

File: ZaloBotServer/src/main/java/com/vn/insee/zalobot/common/status/ImageStatus.java
package com.vn.insee.zalobot.common.status;

public enum ImageStatus {
    WAITING_APPROVAL(1), APPROVED(2), REJECT(3) ;
    private int status;

    ImageStatus(int status) {
        this.status = status;
    }

    public static ImageStatus findByStatus(int status) {
        switch (status) {
            case 1 : return WAITING_APPROVAL;
            case 2 : return APPROVED;
            case 3 : return REJECT;
            default: return null;
        }
    }



    public int getStatus() {
        return status;
    }

    public void setStatus(int status) {
        this.status = status;
    }
}
